python/lsst/pipe/tasks/healSparseMapping.py

```diff
@@ -684,3 +687,232 @@ def _compute_nside_coverage_tract(self, tract_info):
+            doc=f"Minumum-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_consolidated_map_min",
+            storageClass="HealSparseMap",
+            dimensions=("skymap", "band"),
+        )
+        vars()[f"{name}_map_max"] = pipeBase.connectionTypes.Input(
+            doc=f"Maximum-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_map_max",
+            storageClass="HealSparseMap",
+            dimensions=("tract", "skymap", "band"),
+            multiple=True,
+            deferLoad=True,
+        )
+        vars()[f"{name}_consolidated_map_max"] = pipeBase.connectionTypes.Output(
+            doc=f"Minumum-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_consolidated_map_max",
+            storageClass="HealSparseMap",
+            dimensions=("skymap", "band"),
+        )
+        vars()[f"{name}_map_mean"] = pipeBase.connectionTypes.Input(
+            doc=f"Mean-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_map_mean",
+            storageClass="HealSparseMap",
+            dimensions=("tract", "skymap", "band"),
+            multiple=True,
+            deferLoad=True,
+        )
+        vars()[f"{name}_consolidated_map_mean"] = pipeBase.connectionTypes.Output(
+            doc=f"Minumum-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_consolidated_map_mean",
+            storageClass="HealSparseMap",
+            dimensions=("skymap", "band"),
+        )
+        vars()[f"{name}_map_weighted_mean"] = pipeBase.connectionTypes.Input(
+            doc=f"Weighted mean-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_map_weighted_mean",
+            storageClass="HealSparseMap",
+            dimensions=("tract", "skymap", "band"),
+            multiple=True,
+            deferLoad=True,
+        )
+        vars()[f"{name}_consolidated_map_weighted_mean"] = pipeBase.connectionTypes.Output(
+            doc=f"Minumum-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_consolidated_map_weighted_mean",
+            storageClass="HealSparseMap",
+            dimensions=("skymap", "band"),
+        )
+        vars()[f"{name}_map_sum"] = pipeBase.connectionTypes.Input(
+            doc=f"Sum-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_map_sum",
+            storageClass="HealSparseMap",
+            dimensions=("tract", "skymap", "band"),
+            multiple=True,
+            deferLoad=True,
+        )
+        vars()[f"{name}_consolidated_map_sum"] = pipeBase.connectionTypes.Output(
+            doc=f"Minumum-value map of {name}",
+            name=f"{{coaddName}}Coadd_{name}_consolidated_map_sum",
+            storageClass="HealSparseMap",
+            dimensions=("skymap", "band"),
+        )
+
+    def __init__(self, *, config=None):
+        super().__init__(config=config)
+
+        # Remove connections from those that are not configured
```

Comment:
Why would there be unconfigured connections? Is this something a user should be warned about?

Reply:
I've improved the comments here and when I used the same pattern for the tract-based maps.  But the idea is that we have to create all possible connections that are in the registry, and then remove those that weren't configured to run to leave the subset of connections that we want to use.